### PR TITLE
PE-6462: phase 5: report assessments and not-connected staking page

### PR DESCRIPTION
**Pull request by kunstmusik**

### CODE REVIEW

File: src/components/AssessmentDetailsPanel.tsx [hunk 1]
```diff
@@ -119,9 +126,48 @@ const ArNSAssessmentPanel = ({
...
-              <TimerIcon className="size-4" />
-              <div>Total: {arnsAssessment.timings.total} ms</div>
+            <div className="flex flex-col gap-1 px-3 pb-3">
+              <div className="flex gap-1">
+                <TimerIcon className="size-4" />
+                <div className="text-high">
+                  Total: {arnsAssessment.timings.total} ms
+                </div>
+              </div>
+              {arnsAssessment.timings.dns !== undefined && (
```
Comment (atticusofsparta) [inline src/components/AssessmentDetailsPanel.tsx:136]: Seems like theres some opportunity to DRY this a bit. Repeated use of the object key to display the object value in repeated div tags of the same class is good use case for an entries().map

Comment (kunstmusik) [reply src/components/AssessmentDetailsPanel.tsx]: updated code

File: src/pages/Staking/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <div
+            className="invisible size-full rounded-xl
+       bg-gradient-to-r from-gradient-primary-start to-gradient-primary-end p-px group-hover:visible"
+          >
+            <div className="size-full overflow-hidden rounded-xl bg-grey-800"></div>
+          </div>
+          <div className="absolute top-0 z-10 flex size-full flex-col items-center justify-center bg-transparent py-[24px] align-middle">
+            <div className="flex items-center gap-[8px]">
+              <ObserversConnectIcon />
+              <div className="text-gradient">Do you even wallet bro?</div>{' '}
```
[the remark this answers is not in the document]
Comment (kunstmusik) [reply src/pages/Staking/Banner.tsx]: Updated with new text

File: src/pages/Staking/Banner.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            </div>
+
+            <div className="pt-[8px] text-sm text-low">
+              Connect a wallet to splash some cash.
```
[the remark this answers is not in the document]
Comment (kunstmusik) [reply src/pages/Staking/Banner.tsx]: Updated with new text

File: src/pages/Report/GatewayAssessmentsTable.tsx [hunk 4]
```diff
@@ -1,7 +1,9 @@
+import { AoGatewayWithAddress } from '@ar.io/sdk/web';
 import AddressCell from '@src/components/AddressCell';
+import AssessmentDetailsPanel from '@src/components/AssessmentDetailsPanel';
 import Bubble from '@src/components/Bubble';
 import TableView from '@src/components/TableView';
-import { ReportData } from '@src/types';
+import { Assessment, ReportData } from '@src/types';
```
[the remark this answers is not in the document]
Comment (kunstmusik) [reply src/pages/Report/GatewayAssessmentsTable.tsx:6]: Could be handy, might also find it useful to have a loadReport() method in the SDK that handles gunzip and also verification of report data format before returning the data to the caller.

**Review by atticusofsparta — APPROVED** [review]

**Pull request merged**